Add deploy.sh and simplify .travis.yml

stephen-soltesz opened the pull request:
This change adds a `deploy.sh` script that replaces the complext directive from `.travis.yml`. The deploy script performs the same steps as before -- copying files and build artifacts to GCS -- in a more managable and readable form.

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/m-lab/epoxy-images/48)
<!-- Reviewable:end -->

gfr10598 commented:
Maybe "deploy-epoxy-components.sh" ?
Would like a more descriptive script name, but otherwise LGTM.
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/> 

---

Review status: 0 of 2 files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://beta.reviewable.io/reviews/m-lab/epoxy-images/48#-:-LATatO15VrANRXxvrbH:b-hobaj0)*
<!-- Sent from Reviewable.io -->

gfr10598 commented:
Reviewed 2 of 2 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved, some commit checks pending.

---



*Comments from [Reviewable](https://beta.reviewable.io/reviews/m-lab/epoxy-images/48)*
<!-- Sent from Reviewable.io -->

stephen-soltesz commented:
Renamed file, also made it executable (forgot about that), fixed one bug re: the travis env.

---

Review status: 0 of 2 files reviewed at latest revision, all discussions resolved, all commit checks successful.

---



*Comments from [Reviewable](https://beta.reviewable.io/reviews/m-lab/epoxy-images/48#-:-LAURjfmAHHtfUitMPVs:b61oahq)*
<!-- Sent from Reviewable.io -->
